tests/regression/roles/sonic_sflow/tasks/preparation_tests.yml

```diff
@@ -0,0 +1,12 @@
+- name: "initialize sflow test environment"
+  vars:
+    ansible_connection: network_cli
+  sonic_config:
```

Comment:
What is the intended purpose of this line and the next one? (They currently have no effect.)

It looks like they should be deleted.

Reply:
yep, with the comment about Ethernet interfaces, this isn't needed anymore

Reply:
This comment has been addressed in the most recent commit.

Resolved.